views/010_topology.sql

```diff
@@ -45,9 +45,23 @@ WITH
       LEFT JOIN teams ON team_components.team_id = teams.id
     GROUP BY
       team_components.component_id
+  ),
+  log_selectors AS (
```

Comment:
Can we not use jsonb_path_query for this ?

Reply:
changed it to use `jsonb_path_query`.

But turns out the formed `logs` was incorrect. We currently have `["log-selector", "log-selector-2"]` but we want `[{"name": "log-selector"}, {"name": "log-selector-2"}]`